Firestore/core/src/firebase/firestore/local/local_serializer.cc

```diff
@@ -142,7 +142,7 @@ google_firestore_v1_Document LocalSerializer::EncodeDocument(
   }
 
   result.update_time = rpc_serializer_.EncodeVersion(doc.version());
-
+  result.has_update_time = true;
```

Comment:
Do we have any convention on how to pair these up? Should we adopt a convention like set the `has_` field just before/after assigning to the field?

Reply:
I am not aware of any conventions, and no strong preference, costa thinks before is better, just so it's harder to forget to do so.